go/types/opcache.go

```diff
@@ -335,10 +335,10 @@ func decodeValue(bs []byte, asValue bool, vr ValueReader) ([]byte, Value) {
 }
 
 // Note that, if 'v' are prolly trees, any in-memory child chunks will be written to vw at this time.
```

Comment:
Is this comment still true? I'm not entirely sure how it could be, and that would be a problem, no?